code/controllers/ShortcodableController.php

```diff
@@ -6,19 +6,17 @@
  **/
 class ShortcodableController extends LeftAndMain
 {
-    /**
-     * @var string
-     */
-    const URLSegment = 'ShortcodableController';
+    private static $url_segment = 'shortcodes';
+    private static $menu_title = 'Shortcodes';
+    private static $required_permission_codes = 'CMS_ACCESS_AssetAdmin';
```

Comment:
@jelicanin Aah yup thanks. I still think `CMS_ACCESS_LeftAndMain` is more appropriate because this module does not rely on CMS as a dependency. `CMS_ACCESS_CMSMain` is defined in the cms module. Also in some cases users may only be granted access to a particular model admin view, where they can edit content in HTMLEditorFields, but may not have `CMS_ACCESS_CMSMain` permission.